### ISSUE TITLE
youtube player stuck on loading on S20

### ISSUE THREAD

**Issue by smeet84** [PierfrancescoSoffritti/android-youtube-player#630]
Hello, we tested the library on a lot of low end phones and it works. Surprisingly , it doesnt work on S20 samsung and just shows loading with blank screen. Any help shall be appreciated.

**Comment by smeet84** [comment]
we checked the issue and saw in some cases we do not go to Onready state right away, it takes time becuase of which video is stuck on loading else, it plays instantly. Any work around here?

**Comment by crearo** [comment]
+1 struggling with exactly this same.

**Comment by rugovit** [comment]
+1 our production app is having the same issue. Really important to have this issue fixed

**Comment by PierfrancescoSoffritti** [comment]
Is it not working for every video? have you tried taking a random video from the YouTube trending page or something?

**Comment by smeet84** [comment]
anyone could try this for ohter videos?

**Comment by crearo** [comment]
I don't think this is choice of video dependent if so many people are complaining about it!

**Comment by PierfrancescoSoffritti** [comment]
I can't say much about it with so little information. It never happened to me, testing on multiple devices.

**Comment by sjmach** [comment]
Hello, I faced same issue. Please make sure you have closed all webviews before you start a video. This fixed my issue.

**Comment by PierfrancescoSoffritti** [comment]
You can also try to [initialize directly with the video id](https://github.com/PierfrancescoSoffritti/android-youtube-player?tab=readme-ov-file#initialization) to see if that helps.
